src/main/java/com/salesforce/storm/spout/dynamic/kafka/Consumer.java

```diff
@@ -464,6 +478,7 @@ private void handleOffsetOutOfRange(OffsetOutOfRangeException outOfRangeExceptio
 
             // If this partition was out of range
             // we simply log an error about data loss, and skip them for now.
+            // TODO invert this if statement and call continue.
```

Comment:
Lets make this change in a followup PR to keep this PR as simple as possible

Reply:
K, I want to retool what's beneath this anyhow.